src/styles/product-cards.scss

```diff
@@ -0,0 +1,114 @@
+.product-cards {
+  font-family: $main-font-family;
+  font-weight: 400;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  background-color: $background-color;
+
+  &__photo {
+    background-image: url(../images/imac.jpeg);
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-decoration: none;
+    color: $titel-text-color;
+    width: 166px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    text-decoration: none;
+    color: $second-text-color;
+    margin-top: 4px;
+    align-self: self-start;
+  }
+
+  &__botton {
+    display: flex;
+    box-sizing: border-box;
+    width: 100%;
+    height: 40px;
+    background-color: $main-color;
+    border-radius: 5px;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: $background-color;
+    margin-top: 16px;
+
+    &:hover {
+      box-sizing: border-box;
+      color: $main-color;
+      background-color: $background-color;
+      border: 1px solid $main-color;
+    }
+  }
+}
+
+.range {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+  margin-top: 16px;
+
+  &__reviews {
+    display: flex;
+    font-size: 10px;
+    line-height: 14px;
+    color: $titel-text-color;
+    align-items: flex-end;
+  }
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  background-image: url(../images/star.svg);
+}
+
+.stars :nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
<img width="510" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/e0779e5d-bd21-4dd0-af0b-6d4f5a08a598">
paste all styles from the stars task here